style.css

```diff
@@ -0,0 +1,104 @@
+body {
+font-family: 'Oswald', sans-serif;
+}
+
+.top_box {
+box-sizing: border-box;
+background-color: white;
+margin: auto;
+margin-bottom: 40px;
+margin-top: 40px;
+width: 910px;
+height:80px;
+padding: 20px;
+border-top: 1px solid black;
+border-bottom: 10px double rgba(70,45,200,0.7);
+}
+
+.main_box {
+display: flex;
+gap:2px;
+box-sizing: border-box;
+margin: auto;
+width: 910px;
+height:100%;
+}
+
+.category_box {
+background-image: url("https://cdn.galleries.smcloud.net/t/galleries/gf-9fcC-tNGL-eqSc_czekolada-jpg-664x442-nocrop.jpg");
+background-repeat: no-repeat;
+background-size: 300px 150px;
+box-sizing: border-box;
+width: 300px;
+height:310px;
+border: 1px solid black;
+border-radius: 5px;
+}
+.category_box:nth-child(1) {
+background-image: url("https://naszedrinki.pl/wp-content/uploads/2020/04/glowne_alkohole-swiata.jpeg");
+}
+.category_box:nth-child(2) {
+background-image: url("https://cdn.galleries.smcloud.net/t/galleries/gf-9fcC-tNGL-eqSc_czekolada-jpg-664x442-nocrop.jpg");
+}
+.category_box:nth-child(3) {
```

Comment:
uważałabym z bazowaniem na strukturze w stylach, jak ktoś zamieni kolejnościa 2 boxy, to rozjada Ci sie obrazki z zawartościa boxa